Example Apps/Example Swift/MenuManager.swift

```diff
@@ -10,12 +10,29 @@ import Foundation
 import SmartDeviceLink
 import SmartDeviceLinkSwift
 
+typealias RefreshTemplateHandler = ((SDLPredefinedLayout) -> Void)
+
 class MenuManager: NSObject {
+    fileprivate var refreshTemplateHandler: RefreshTemplateHandler?
+
+    /// Setting current selected template
+    public fileprivate(set) var currentTemplate: SDLPredefinedLayout {
+        didSet {
+            guard let refreshTemplateHandler = refreshTemplateHandler else { return }
+            refreshTemplateHandler(currentTemplate)
+        }
+    }
+
+    init(updateTemplate: RefreshTemplateHandler? = nil) {
```

Comment:
```suggestion
    init(sdlManager: SDLManager, refreshTemplateHandler: RefreshTemplateHandler? = nil, choiceSetManager: PerformInteractionManager) {
```

Reply:
If the `sdlManager` is passed in the `init`, we can get rid of all the passed `sdlManager`s for each menu item.